### PR TITLE
Google Analytics V2 Cloud Extract  doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Create Google Analytics V2 Cloud Extract  doc

### CODE REVIEW

File: docs/sources/extract/google-analytics-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+The rest of the settings are as follows:
+<img src="../../assets/cloud-extract-sources/ga-oauth-settings-2.png" alt="Google Analytics credentials" />
+<img src="../../assets/cloud-extract-sources/ga-oauth-settings-3.png" alt="Google Analytics credentials" />
+
+- **Replication Start Date**: Choose the start date from which you want RudderStack to ingest the Google Analytics data. RudderStack will not replicate any data before this date.
+- **View ID**: Enter the ID of the Google Analytics View you want to fetch the data from. Refer to this [documentation](https://docs.acquia.com/customer-data-platform/connectors/web-analytics-provider/google-analytics-profileid/) for more information on obtaining the View ID.
+- **Custom Reports (Optional)**: Use this field to sync your [custom reports](https://support.google.com/analytics/answer/1033013?hl=en) from Google Analytics. Click **Add Report** and enter the **Report Name**, **Metrics**, and **Dimensions** which RudderStack will add as a JSON array, for example: `{"name": string, "dimensions": [string], "metrics": [string]}`.
```
Comment (am6010) [inline docs/sources/extract/google-analytics-v2.mdx:51]: They can also edit an existing report or delete. They can add many reports. 
Also I don't think they need to know the underneath implementation of json array.

**Review by am6010 — COMMENTED** [review]
Looks good ! I have a minor comment.

File: docs/sources/extract/google-analytics-v2.mdx (hunk 1 above)
Comment (ishachopra0401) [reply docs/sources/extract/google-analytics-v2.mdx]: Thanks for highlighting this, @am6010!
I have added these in the doc.

**Review by am6010 — APPROVED** [review]
@ishachopra0401  thank you for the changes!

File: docs/sources/extract/google-analytics-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+title: "Google Analytics V2"
+description: Step-by-step guide on ingesting data from Google Analytics into RudderStack.
+aliases: ["Google Analytics Extract Source", "Singer Google Analytics"]
+---
+
+[Google Analytics](https://analytics.google.com/analytics/web/#/) is one of the popular analytics service that allows you to track and report your website traffic across a variety of sources.
```
Comment (ameypv-rudder) [inline docs/sources/extract/google-analytics-v2.mdx:7]:
```suggestion
[Google Analytics](https://analytics.google.com/analytics/web/#/) is a popular analytics service that lets you track and report your website traffic across a variety of sources.
```

File: docs/sources/extract/google-analytics-v2.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
+2. Go to **Sources** > **New source** > **Cloud Extract** and select **Google Analytics V2** from the list of sources.
+3. Assign a name to your source and click **Continue**.
+
+### Connection settings
+
+Next, configure the following dashboard settings:
+
+- **Credentials**: Choose from the two authentication types as explained below:
```
Comment (ameypv-rudder) [inline docs/sources/extract/google-analytics-v2.mdx:34]:
```suggestion
- **Credentials**: Choose from one of the following two authentication types:
```

File: docs/sources/extract/google-analytics-v2.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+    - **Authenticate via Google (Oauth)**: Authenticate your Google Analytics account via Google (Oauth) by clicking
```
Comment (ameypv-rudder) [inline docs/sources/extract/google-analytics-v2.mdx:36]:
```suggestion
    - **Authenticate via Google (Oauth)**: Authenticate your Google Analytics account via Google (OAuth) by clicking
```

**Pull request merged**